test2code/src/main/kotlin/com/epam/drill/test2code/coverage/CoverageSender.kt

```diff
@@ -30,7 +30,10 @@ interface CoverageSender {
     fun stopSendingCoverage()
 }
 
-private val COVERAGE_RETENTION_LIMIT_BYTES = BigInteger.valueOf(JvmModuleConfiguration.getCoverageRetentionLimit())
+private val COVERAGE_RETENTION_LIMIT_BYTES = DataSize.parse(JvmModuleConfiguration.getCoverageRetentionLimit())
```

Comment:
1. No reason to use uppercase
2. Move `DataSize.parse` to `AgentArguments.kt` and just use `JvmModuleConfiguration.getCoverageRetentionLimit()` in `IntervalCoverageSender` initialization

Reply:
convert to BigInteger

Reply:
done